runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsOverlay.java

```diff
@@ -38,6 +38,8 @@
 import net.runelite.client.util.ColorUtil;
 import net.runelite.client.util.ImageUtil;
 
+import static net.runelite.api.widgets.WidgetID.GRAVESTONE_GROUP_ID;
```

Comment:
Move this import between the `javax.inject.Inject` and `net.runelite.api.widgets.WidgetItem` imports; we don't separate static and non-static imports.

Reply:
Done !